biz.aQute.bndlib/src/aQute/bnd/osgi/PluginsContainer.java

```diff
@@ -388,4 +399,9 @@ protected void close() {
 		plugins.clear();
 		missingCommand.clear();
 	}
+
+	public Map<Class<?>, Attrs> getInterfaces() {
+		return interfaces;
```

Comment:
For thread safety, this method should probably return an unmodifiable view of the map.